File: baml_language/sdk_tests/crates/rust/type_shapes/customizable/roundtrip_tests/test_void.rs
//! Roundtrip coverage for `baml_sdk::void` — `void` return lowers to `()`.

use baml_sdk::void::no_op;

#[test]
fn test_void_no_op() {
    // `-> void` lowers to `()`; the successful unwrap is the `is None`
    // assertion.
    no_op().unwrap();
}

File: baml_language/sdk_tests/crates/swift/type_shapes/customizable/roundtrip_tests/TestVoid.swift
// Roundtrip coverage for `Baml.void` — port of python_pydantic2
// `roundtrip_tests/test_void.py`. `void` return lowers to Swift `Void`.
import XCTest
import Baml

final class TestVoid: XCTestCase {
    func test_void_no_op() throws {
        try Baml.void.no_op()
    }
}
